coral/src/domain/schema-request/schema-request-api.ts

```diff
@@ -55,7 +66,35 @@ const getSchemaRequestsForApprover = (
         queryObject
       ).toString()}`
     )
-    .then(transformGetSchemaRequestsForApproverResponse);
+    .then(transformGetSchemaRequests);
+};
+
+type GetSchemaRequestsQueryParams = ResolveIntersectionTypes<
+  Required<Pick<KlawApiRequestQueryParameters<"getSchemaRequests">, "pageNo">> &
```

Comment:
Is this `Required` utility type needed? I think `pageNo` is already required?

![Screenshot 2023-03-15 at 10 47 33](https://user-images.githubusercontent.com/20607294/225271993-1357de20-81c7-4549-a040-257ed5590ccd.png)

I think what we are trying to do here is exclude the `currentPage` property we are not using, so maybe we can use `Omit` and achieve the samne result?

![Screenshot 2023-03-15 at 10 49 42](https://user-images.githubusercontent.com/20607294/225272264-21b00151-a26f-4df9-b8de-4d51d30a56c1.png)

Reply:
`pageNo` is the parameter we're using in the API call, not currentPage. 
But yes, it's required anyway, seems I overlooked that. Will update!